runtime/src/prioritization_fee_cache.rs

```diff
@@ -543,9 +568,18 @@ mod tests {
             let lock = prioritization_fee_cache.cache.read().unwrap();
             let fee = lock.get(&slot).unwrap();
             assert_eq!(2, fee.get_min_transaction_fee().unwrap());
-            assert!(fee.get_writable_account_fee(&write_account_a).is_none());
-            assert_eq!(5, fee.get_writable_account_fee(&write_account_b).unwrap());
-            assert!(fee.get_writable_account_fee(&write_account_c).is_none());
+            assert_eq!(
+                2,
+                fee.get_min_writable_account_fee(&write_account_a).unwrap()
+            );
```

Comment:
this says the information stored is irrelevant: landing into block requires at least 2, implies write locking account_a would need >=2. so storing 2 for account_a has no additional information but wasting space.